[DOCS] Rules page UI updates (#1450)
elastic/security-docs: docs/detections/rules-ui-monitor.asciidoc

[role="screenshot"]
image::images/monitor-table.png[]

> For detailed information on a rule, its generated alerts, and its errors, click on a rule name in the *All rules* table. You can also perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.

Reviewer: ```suggestion
For detailed information on a rule, alerts it generated, and associated errors, click on its name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
```
Nits. Mostly I just thought that "its generated alerts" was an awkward turn of phrase.

Author: Agree, with a couple slight amendments:
```suggestion
For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
```

> For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.

Reviewer 2: To me, it looks like, `This allows also..` in 
> For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions...

refers more to what is possible after clicking on rule's name rather that what is possible to do on monitoring table.
Maybe, worth to highlight actions are possible on monitoring table?

Author: @vitaliidm I agree. We could reorder and rephrase slightly to lead with those actions:
```suggestion
You can perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules. For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. 
```

Reviewer 2: Thanks, @joepeeples. Looks good
